[Bug]: container-build-test pipeline for gpu-reset is flaky due to "Too many requests" in docker.io

### Prerequisites

- [x] I searched existing issues
- [x] I can reproduce this issue

### Bug Description

https://github.com/NVIDIA/NVSentinel/actions/runs/21671263818/job/62479486484

Can't think of any quick way to fix this, marking this as low priority



### Component

Other

### Steps to Reproduce

Trigger pipelines with high rates

### Environment

- NVSentinel version: 0.8.0
- Kubernetes version:
- Deployment method:


### Logs/Output

```
[6](https://github.com/NVIDIA/NVSentinel/actions/runs/21671263818/job/62479486484#step:7:428)
WARNING: No output specified with docker-container driver. Build result will only remain in the build cache. To push result image into registry use --push or to load image into docker use --load
Dockerfile:15
--------------------
  13 |     # limitations under the License.
  14 |     
  15 | >>> FROM nvidia/cuda:13.1.0-runtime-ubuntu24.04
  16 |     
  17 |     COPY gpu-reset/gpu_reset.sh /usr/local/bin/gpu_reset.sh
--------------------
ERROR: failed to build: failed to solve: nvidia/cuda:13.1.0-runtime-ubuntu24.04: failed to resolve source metadata for docker.io/nvidia/cuda:13.1.0-runtime-ubuntu24.04: failed to copy: httpReadSeeker: failed open: unexpected status from GET request to https://registry-1.docker.io/v2/nvidia/cuda/manifests/sha256:17b999b6882af79f41995f4807c32335734434d91ff0f6cdeaabc9115e7dab64: 429 Too Many Requests
toomanyrequests: You have reached your unauthenticated pull rate limit. https://www.docker.com/increase-rate-limit
make: *** [../make/docker.mk:28: docker-build] Error 1
make: Leaving directory '/home/runner/_work/NVSentinel/NVSentinel/gpu-reset'
Error: Process completed with exit code 2.
```

--- a/gpu-reset/Dockerfile
+++ b/gpu-reset/Dockerfile
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM nvr.io/nvidia/cuda:13.1.0-runtime-ubuntu24.04
+FROM nvcr.io/nvidia/cuda:13.1.0-runtime-ubuntu24.04
 
 COPY gpu-reset/gpu_reset.sh /usr/local/bin/gpu_reset.sh
 

--- a/gpu-reset/Dockerfile
+++ b/gpu-reset/Dockerfile
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM nvidia/cuda:13.1.0-runtime-ubuntu24.04
+FROM nvr.io/nvidia/cuda:13.1.0-runtime-ubuntu24.04
 
 COPY gpu-reset/gpu_reset.sh /usr/local/bin/gpu_reset.sh
 

